src/SFA.DAS.Commitments.Database/StoredProcedures/GetPendingTransferRequests.sql

```diff
@@ -0,0 +1,19 @@
+﻿CREATE PROCEDURE [dbo].[GetPendingTransferRequests]
+AS
+
+SELECT 
```

Comment:
Is it worth considering this query wrt the db indexes?
Perhaps add Status to the existing index containing CommitmentId on the TransferRequest table?
If this warrants changing the indexes then perhaps check its execution plan.